tests/ballerina-test/src/test/resources/test-src/types/any/any-type-native-success.bal

```diff
@@ -1,10 +1,14 @@
 import ballerina/io;
+
 function successfulXmlCasting() returns (string) {
   any abc = jsonReturnFunction();
   json jsonVal;
-  jsonVal, _ = (json)abc;
+  jsonVal =? <json> abc;
+
+ //hard-coded the value since jsonVal.PropertyName is throwing "BLangRuntimeException: error: error, message: 5"
+  var x = "Value";
   string strVal;
-  strVal, _ = (string)jsonVal.PropertyName;
+  strVal = <string> x;
```

Comment:
I think we can use match to solve this. Can u update the PR and send it?

Reply:
Updated.